open-metadata-implementation/adapters/open-connectors/connector-configuration-factory/pom.xml

```diff
@@ -55,6 +55,16 @@
             <artifactId>audit-log-console-connector</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.odpi.egeria</groupId>
+            <artifactId>audit-log-log4j-connector</artifactId>
```

Comment:
I have verified the dependency tree and don't see any regressions so far (beyond the logging  from the archive builder, which is new rather than a regression & is dealt with under #2321 )